python/activity_stream/activity_stream.py

```diff
@@ -158,6 +159,30 @@ def note_widget(self):
         """
         return self.ui.note_widget
 
+    def _get_clickable_user_icons(self):
+        """
+        Whether user icons in the activity stream display as clickable.
+        If True, a pointing hand cursor will be shown when the mouse is
+        hovered over the icons, otherwise the default arrow cursor will be
+        used.
+        """
+        return self._clickable_user_icons
+
+    def _set_clickable_user_icons(self, state):
+        self._clickable_user_icons = bool(state)
+
+        for widget in self._activity_stream_data_widgets.values():
+            if isinstance(widget, NoteWidget):
```

Comment:
what does it mean that it only applies to NoteWidgets? Should this be reflected in the docs?